controllers/survey.controller.js

```diff
@@ -15,15 +18,86 @@ async function getSurveyData(user_id) {
     });
 }
 
+
+const getdata = (request, response) => {
```

Comment:
@sonalratho26 
Name this variable that represents what kind of data it holds. In this case `getData` will be a global naming convension

Reply:
okay sir i have changed name